test/data-ingest/data-ingest.py

```diff
@@ -0,0 +1,508 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
I would suggest a separate .py file for each base class and its descendents.

Reply:
Yes, this was just for a quick PoC to see if this makes sense, will split it up as I work on it.